File: packages/work/take-picture/src/main/java/com/sofar/take/picture/ui/MainActivity.java
package com.sofar.take.picture.ui;

import android.os.Bundle;

import androidx.annotation.Nullable;

import com.sofar.base.BaseActivity;
import com.sofar.base.R;

public class MainActivity extends BaseActivity {

  @Override
  protected void onCreate(@Nullable Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_container);

    getSupportFragmentManager().beginTransaction()
      .replace(R.id.fragment_container, new MainFragment())
      .commitAllowingStateLoss();
  }


}

File: packages/wan-android/src/main/java/com/sofar/wan/android/MainActivity.kt
package com.sofar.wan.android

import android.os.Bundle
import com.sofar.core.ui.activity.BaseUIActivity
import com.sofar.core.legacy.R as legacyR

class MainActivity : BaseUIActivity() {

  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    setContentView(legacyR.layout.activity_container)

    supportFragmentManager.beginTransaction()
      .replace(legacyR.id.fragment_container, MainFragment())
      .commitAllowingStateLoss()
  }

}